src/ui/StackedBarChart.js

```diff
@@ -586,7 +599,7 @@ WGL.ui.StackedBarChart = function(m, div_id, x_label, filterId, params) {
     }
 
 
-
+    // nepouziva se !!!!!
     function brushO() {
 
```

Comment:
Avoid comments in  Czech :) Feel free to delete that if it is not useful anymore